scripts/security/check_secret_expiration.py

```
@@ -0,0 +1,584 @@
+            f"{days_until} days | {secret.owner} | {status} |"
+        )
+
+    return "\n".join(lines)
+
+
+def generate_github_report(check_type: str, days: int = 7) -> str:
+    """Generate GitHub-formatted Markdown report."""
+    report_lines = [
+        "# Secret Rotation Status Report",
+        "",
+        f"**Generated:** {datetime.now().strftime('%Y-%m-%d %H:%M:%S UTC')}",
```

Ruff (0.14.4)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)